docs/tutorials/getting-started.md

```diff
@@ -6,34 +6,34 @@ If you have any questions about anything related to LDK, feel free to ask our co
 
 ## System Requirements
 MacOS, Windows and Linux are supported.
-
 ## Installation
 To add LDK to a project, run:
 
-:::: tabs
-
-::: tab Rust
-```toml
-
-# Add the following dependencies to your cargo.toml and replace {VERSION} with the version number you want to use.
-
-[dependencies]
-lightning = { version = {VERSION}, features = ["max_level_trace"] }
-lightning-block-sync = { version = {VERSION}, features = [ "rpc-client" ] }
-lightning-invoice = { version = {VERSION} }
-lightning-net-tokio = { version = {VERSION} }
-lightning-persister = { version = {VERSION} }
-lightning-background-processor = { version = {VERSION} }
-lightning-rapid-gossip-sync = { version = {VERSION} }
-```
-:::
-
-::: tab Java
+<CodeSwitcher :languages="{rust:'Rust', java:'Java', kotlin:'Kotlin', javascript:'JavaScript'}">
+  <template v-slot:rust>
+ 
+  ```toml
```

Comment:
Indentation levels seem inconsistent across the first `CodeSwitcher` block, FWIW. The second block seems fine though with two spaces and one within the formatted language blocks.